backend/src/version/v0_3_3_1.rs

```diff
@@ -61,6 +61,8 @@ impl VersionT for Version {
                 .put(db, &parsed_url)
                 .await?;
         }
+        ui["theme"] = json!("Dark".to_string());
+        ui["widgets"] = json!([]);
```

Comment:
I believe that I had a fix to save the migration, hopefully I don't forget this migration for the big refactor we are in.

Reply:
:notebook: These are notes for me, but the code looks great. I'm going to have to merge this into the big refactor